.template-lintrc.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  extends: 'recommended',
+  rules: {
+    'no-curly-component-invocation': false,
+    'no-implicit-this': false,
```

Comment:
```suggestion
    'no-curly-component-invocation': 'warn',
    'no-implicit-this': 'warn',
```

Reply:
Good catch! I updated the template and removed the rules.